addons/point_of_sale/static/src/js/Misc/NumberBuffer.js

```diff
@@ -226,7 +226,7 @@ odoo.define('point_of_sale.NumberBuffer', function(require) {
             };
             if (input === undefined || input === null) return;
             let isFirstInput = isEmpty(this.state.buffer);
-            if (input === this.decimalPoint) {
+            if (input === this.decimalPoint || input === '.') {
```

Comment:
it works in the case decimal point is ',', but if decimal point is '.' the ', ' won't work. Don't you think we should just check that if it is '.' or ',' it will trigger the decimal

Reply:
Do you mean changing the condition into: `input === ',' || input === '.'`?

Reply:
yes if it looks good to you

Reply:
> Don't you think we should just check that if it is '.' or ',' it will trigger the decimal
I'd say yes to this, that we should check the input if it is `.` or `,` and interpret it as decimal.

Reply:
It actually is. :)

Reply:
it is not if this.decimalPoint = '.'

Reply:
Oops sorry. I meant I'm okay with this response of yours:
> yes if it looks good to you

I forced push the change.